src/browser/api/window.js

```diff
@@ -57,6 +57,7 @@ import {
     DEFAULT_RESIZE_REGION_BOTTOM_RIGHT_CORNER,
     DEFAULT_RESIZE_SIDES
 } from '../../shapes';
+import { windowSetBoundsToVisible } from '../utils';
```

Comment:
Why is this a separate line from the previous utils import of `clipBounds`?

Reply:
fixed 👍